### ISSUE TITLE
LR Scheduler

### ISSUE THREAD

**Issue by kakumarabhishek** [mgrankin/over9000#13]
Referring to https://github.com/mgrankin/over9000/issues/1, how do I use the various LR schedules in PyTorch? Can you please provide an example?

[These](https://pytorch.org/docs/stable/search.html?q=lr+schedule&check_keywords=yes&area=default) are the LR schedulers that are available in PyTorch. For example, how would I implement the "Flat and anneal" LR schedule? (I have not used fastai, so any help will be appreciated).

Thank you.

**Comment by pabloppp** [comment]
@kakumarabhishek I have made an implementation of the Flat & anneal scheduler here: https://github.com/pabloppp/pytorch-tools/blob/master/torchtools/lr_scheduler/delayed.py

What I actually do is delay the start of the annealing for N epochs.
You can install my repository using pip and use my implementation pretty easily if you want
```python
from torchtools.lr_scheduler import DelayedCosineAnnealingLR
scheduler = DelayedCosineAnnealingLR(optimizer, 50, 50) # this will have a flat lr for 50 epochs then cosine annealing for another 50 epochs
```

**Comment by wangg12** [comment]
@kakumarabhishek You can refer to my implementation [https://github.com/wangg12/flat_anneal_scheduler.pytorch](https://github.com/wangg12/flat_anneal_scheduler.pytorch). I've tested on my own task, it is better than simple multi-step lr scheduler.
